apps/api/src/report/report.util.service.ts

```diff
@@ -215,6 +215,101 @@ export class ReportUtilService {
       FROM report WHERE registration_date::date >= '${from}';
     `;
   }
+  /**
+   *
+   * @param mappedStatusesString
+   * @param from start
+   * @param to end
+   * @param BCCNM_NEW_PROCESS
+   * @returns Query string
+   */
+  reportFour(from: string, to: string, BCCNM_NEW_PROCESS: boolean): string {
+    return `
+      SELECT 
+        COUNT(DISTINCT applicant),applicant.status_id 
+        FROM ien_applicants applicant
+        INNER JOIN ien_applicant_status ias ON ias.id = applicant.status_id
+        INNER JOIN ien_applicant_status_audit status_audit 
+          ON applicant.id = status_audit.applicant_id 
+          AND applicant.status_id = status_audit.status_id
+        WHERE
+          applicant.new_bccnm_process = ${BCCNM_NEW_PROCESS ? 'TRUE' : 'FALSE'}
+          AND
+            ias.category = 'IEN Licensing/Registration Process'
+          AND 
+              start_date::date >= '${from}' 
+              AND start_date::date <= '${to}'
+        GROUP BY applicant.status_id`;
+  }
+  /**
+   * Counts the number of applicants with a partial license (RN or LPN)
+   * Exclude applicants who have withdrawn from the program or been hired, or who have a full license
+   * @param BCCNM_NEW_PROCESS
+   * @returns A query string
+   */
+  partialLicenceQuery(BCCNM_NEW_PROCESS: boolean) {
+    return `
+      SELECT 
+        count(distinct status_audit.applicant_id)
+      FROM 
+        "ien_applicant_status_audit" "status_audit" 
+      LEFT JOIN 
+        ien_applicants applicant ON applicant.id = status_audit.applicant_id 
+      WHERE 
+        applicant.new_bccnm_process IS NOT NULL AND applicant.new_bccnm_process = ${
+          BCCNM_NEW_PROCESS ? 'TRUE' : 'FALSE'
+        } AND
+        "status_audit"."status_id" IN ('91F55FAA-C71D-83C8-4F10-3A05E778AFBC','D2656957-EC58-15C9-1E21-3A05E778DC8E' )
+        AND NOT EXISTS (
+          SELECT *
+          FROM 
+            "ien_applicant_status_audit" AS T2
+          WHERE 
+            T2.applicant_id = status_audit.applicant_id
+            AND T2.status_id IN ('f84a4167-a636-4b21-977c-f11aefc486af', '70b1f5f1-1a0d-ef71-42ea-3a0601b46bc2', '632374e6-ca2f-0baa-f994-3a05e77c118a', '18AA32C3-A6A4-4431-8283-89931C141FDE')
```

Comment:
It's hard to know what these IDs mean. Could we use `STATUS` enum or add comments?